docs/connection-pools.md

```diff
@@ -79,6 +79,7 @@ The connection pool instance is also the main point of configuration. Let's take
                    a particular address family. Using `local_address="0.0.0.0"` will
                    connect using an `AF_INET` address (IPv4), while using `local_address="::"`
                    will connect using an `AF_INET6` address (IPv6).
+* `socket_options`: Socket options that have to be included in the TCP socket when the connection was established.
```

Comment:
let's be consistent with our placement of `socket_options` in the list, so that the docs match the code.